conf/layer.conf

```diff
@@ -12,6 +12,6 @@ BBFILE_PRIORITY_iris-thirdparty = "6"
 LAYERDEPENDS_iris-thirdparty = "core \
                                 openembedded-layer"
 
-LAYERSERIES_COMPAT_iris-thirdparty = "dunfell gatesgarth hardknott"
+LAYERSERIES_COMPAT_iris-thirdparty = "honister"
```

Comment:
Sicher das wir hier nur "honister" wollen. Damit sollten wir doch beim Bauen Probleme bekommen, weil noch nicht alles umgestellt ist, oder?

Reply:
This is only to be added in the master branch (which is supposed to be compatible with the poky master), not to the dunfell branch.